app/controllers/navigation/Navigator.scala

```diff
@@ -302,7 +302,8 @@ object Navigator {
   }
 
   val clearanceCacheItemDependent: PartialFunction[DeclarationPage, (ExportsDeclaration, Mode, String) => Call] = {
-    case CommodityMeasure => commodityMeasureClearancePreviousPage
+    case CommodityMeasure   => commodityMeasureClearancePreviousPage
+    case PackageInformation => packageInformationClearancePreviousPage
```

Comment:
I feel like the case for `PackageInformation` should be removed from `clearanceItemPage`, isn't it?

Reply:
You are correct Maciej